WebContent/js/utilities/storageHelper.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+export const ENABLE_REDUX_LOGGER = 'ZoweJes.enableReduxLogger';
+export const NOTIFICATION_DURATION = 'ZoweJes.notificationDuration';
+export const LAST_FILTERS = 'ZoweJes.lastFilters';
+
+
+export function getStorageItem(storageKey) {
+    return window.localStorage.getItem(storageKey);
```

Comment:
```suggestion
    return JSON.parse(window.localStorage.getItem(storageKey));
```
I would suggest to update this getter function with inline JSON parsing.
And then we can remove multiple comparing 'true' to string, parseInt or JSON.parse on every _getStorageItem_ function call. (It will affect Setting.js, Snackbar.js, Filters.js and index.js)

Thanks, Sergei

Reply:
modified